tests/release/service/release_plan_owner_ref.go

```diff
@@ -17,34 +18,48 @@ import (
 var _ = framework.ReleaseServiceSuiteDescribe("[HACBS-2469]test-releaseplan-owner-ref-added", Label("release-service", "releaseplan-ownerref"), func() {
 	defer GinkgoRecover()
 	var fw *framework.Framework
+	var kubeAdminClient *framework.ControllerHub
 	var err error
-	var devNamespace string
+	var devNamespace = "rp-ownerref"
 	var releasePlan *releaseApi.ReleasePlan
 	var releasePlanOwnerReferencesTimeout = 1 * time.Minute
+
+	var testEnvironment = utils.GetEnv("TEST_ENVIRONMENT", releasecommon.UpstreamTestEnvironment)
+
 	AfterEach(framework.ReportFailure(&fw))
 
 	BeforeAll(func() {
-		fw, err = framework.NewFramework(utils.GetGeneratedNamespace("rp-ownerref"))
-		Expect(err).NotTo(HaveOccurred())
-		devNamespace = fw.UserNamespace
+		if testEnvironment == releasecommon.DownstreamTestEnvironment {
+			fw, err = framework.NewFramework(utils.GetGeneratedNamespace(devNamespace))
+			Expect(err).NotTo(HaveOccurred())
+			devNamespace = fw.UserNamespace
```

Comment:
Below sentence need to be added in this block?
`kubeAdminClient = fw.AsKubeAdmin`